tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -743,11 +743,11 @@ public function testFetchAllSupportFetchClass()
 
         $results = $stmt->fetchAll(
             FetchMode::CUSTOM_OBJECT,
-            __NAMESPACE__.'\\MyFetchClass'
+            MyFetchClass::class
         );
 
         self::assertCount(1, $results);
-        self::assertInstanceOf(__NAMESPACE__.'\\MyFetchClass', $results[0]);
+        self::assertInstanceOf(MyFetchClass::class, $results[0]);
```

Comment:
Not strictly needed, but affected by https://github.com/phpstan/phpstan-phpunit/issues/27.